src/Commands/DrupalCommands.php

```diff
@@ -78,45 +78,47 @@ public function setRuntimeConfig(ConsoleCommandEvent $event)
      * @return \Robo\Collection\CollectionBuilder
      */
     public function siteInstall(array $options = [
-      'root'              => InputOption::VALUE_REQUIRED,
-      'base-url'          => InputOption::VALUE_REQUIRED,
-      'site-name'         => InputOption::VALUE_REQUIRED,
-      'site-mail'         => InputOption::VALUE_REQUIRED,
-      'site-profile'      => InputOption::VALUE_REQUIRED,
-      'site-update'       => InputOption::VALUE_REQUIRED,
-      'site-locale'       => InputOption::VALUE_REQUIRED,
-      'account-name'      => InputOption::VALUE_REQUIRED,
-      'account-password'  => InputOption::VALUE_REQUIRED,
-      'account-mail'      => InputOption::VALUE_REQUIRED,
-      'database-user'     => InputOption::VALUE_REQUIRED,
-      'database-password' => InputOption::VALUE_REQUIRED,
-      'database-host'     => InputOption::VALUE_REQUIRED,
-      'database-port'     => InputOption::VALUE_REQUIRED,
-      'database-name'     => InputOption::VALUE_REQUIRED,
-      'sites-subdir'      => InputOption::VALUE_REQUIRED,
-    ])
+            'root' => InputOption::VALUE_REQUIRED,
+            'base-url' => InputOption::VALUE_REQUIRED,
```

Comment:
This indentation is weird, why is this indented with 8 characters? I would expect 4.